src/enhancers/automaton/queue.js

```diff
@@ -0,0 +1,8 @@
+export default (initialValue, { que = 'que', push = 'push' } = {}) => ({
+  init: {
+    [que]: initialValue,
```

Comment:
Should be `queue`, as the filename suggests.

For more information: https://www.merriam-webster.com/dictionary/Que